students/ChelseaSmith/lesson7/html_render.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+from io import StringIO
+
+#  a series of classes used to generate html
+
+
+class Element:
+    tag = ""
+    indent = " "
+
+    def __init__(self, content=None, **kwargs):
+        self.content = content
+        self.kwargs = kwargs
+        if self.content != None:
```

Comment:
`is` is used with object comparison for `None` / `True` / `False` and other objects.